panther_analysis_tool/schemas.py

```diff
@@ -204,7 +204,7 @@
         "AnalysisType": Or("lookup_table"),
         "LookupName": str,
         "Enabled": bool,
-        Or("DataFile", "Refresh"): Or(
+        Or("FileName", "Refresh"): Or(
```

Comment:
not clear on what the implication are around this. When you're free can we have a quick chat in slack/zoom?